sourmash/command_compute.py

```diff
@@ -72,11 +66,11 @@ def compute(args):
                         help="name the signature generated from each file after the first record in the file (default: False)")
     parser.add_argument('--input-is-10x', action='store_true',
                         help="Input is 10x single cell output folder (default: False)")
-    parser.add_argument('--count-valid-reads', default=0, type=int,
+    parser.add_argument('--min-umi-per-barcode', default=0, type=int,
```

Comment:
Changing arguments require major version bumps. In this case this argument is already available in sourmash 2.2.0, so we need to revert it to the old name or adapt the call to bam2fasta (convert a `--count-valid-reads` argument in sourmash to `--min-umi-per-barcode` argument in `bam2fasta` at https://github.com/dib-lab/sourmash/pull/768/files#diff-56040d40281982d44bcd65f91eb0b132R280)